examples/appSpec/zomes/three/code/index.ts

```diff
@@ -18,6 +19,20 @@ function test_debug(val: string): void {
   debug(val);
 }
 
+@serializable
```

Comment:
I think @serializable as a word is a pain. Since we use `@zome_function`, we could use `@can_stringify`

Reply:
Yeah I like that. I will change it to @can_stringify

Reply:
sweet

Reply:
Decorator has been changed to `@can_stringify`. The deserializable one we could call `@can_parse` to match